.github/workflows/build.yml

```diff
@@ -5,34 +5,59 @@ on:
     branches: [ master ]
 
 jobs:
+
   build:
     runs-on: ubuntu-latest
     steps:
-
     - name: Checkout repo
       uses: actions/checkout@v2
       with:
         fetch-depth: 0
-
     - name: Set up Go
       uses: actions/setup-go@v2
       with:
         go-version: ^1.15
-
+    - name: Build binaries
+      run: go run build/build.go
     - name: Run unit tests
       run: go test ./...
 
-    - name: Run integration tests for BIND provider
-      working-directory: integrationTest
-      run: go test -v -verbose -provider BIND
-
-    - name: Run integration test for ROUTE53 provider
+  integration-tests:
+    runs-on: ubuntu-latest
+    needs: build
+    strategy:
+      matrix:
+        provider: [BIND, AZURE_DNS, NAMEDOTCOM, ROUTE53, GCLOUD, CLOUDFLAREAPI]
```

Comment:
Minor nit.  Sort the names of the providers and the list of env variables.

Reply:
ok! https://github.com/StackExchange/dnscontrol/commit/f145ae14386f92a04408851e5f5efaa725e898fe

I originally kept them in the same order as in TC so they would run in the same sequence. But they're running in parallel now, so there's no sequence and this is a tad more readable.